mempool/src/error/mod.rs

```diff
@@ -85,6 +85,8 @@ pub enum MempoolPolicyError {
 pub enum TxValidationError {
     #[error("Chainstate error")]
     ChainstateError(#[from] ChainstateError),
+    #[error("Not accepting transactions during initial block download")]
+    AddedDuringIBD,
```

Comment:
A nitpick: this sounds like not accepting the transaction was the actual problem (like we were supposed to accept it but couldn't). Probably something like "Transaction added during initial block download" would be a bit better, at least it'd be more consistent with other error messages in this file.